udf/json_extract_string_int_map.sql

```diff
@@ -0,0 +1,26 @@
+/*
+
+Returns an array of key/value structs from a string representing a JSON map.
+
+Used by udf_scalar_row.
+
+*/
+
+CREATE TEMP FUNCTION
+  udf_json_extract_string_int_map (input STRING) AS (ARRAY(
```

Comment:
this is why udf_scalar_row is too complicated. this should be a JS function, and `udf_scalar_row` should be removed in favor of putting the content directly in `main_summary.sql`

Reply:
> this should be a JS function

or better yet, something like:
```sql
CREATE TMP FUNCTION
  udf_js_keyed_scalars_row(processes STRING)
  RETURNS STRUCT<
    scalar_parent_telemetry_accumulate_unknown_histogram_keys ARRAY<STRUCT<key STRING, value INT>>,
    scalar_parent_telemetry_event_counts ARRAY<STRUCT<key STRING, value INT>>,
    scalar_content_telemetry_accumulate_unknown_histogram_keys ARRAY<STRUCT<key STRING, value INT>>,
    scalar_content_telemetry_event_counts ARRAY<STRUCT<key STRING, value INT>>>
  LANGUAGE js AS """
...
"""
```

Reply:
turns out just removing excess subquery expressions was sufficient, so this should be fine for now.